src/dashboard/ActivityComponent.vue

```diff
@@ -39,6 +40,7 @@ export default {
     },
   },
   methods: {
+    formatSecs: formatSecs,
```

Comment:
Turn this into a global filter (https://vuejs.org/v2/guide/filters.html) and call it something like: `friendlyDuration`